.circleci/config.yml

```diff
@@ -249,6 +249,41 @@ jobs:
       - store_artifacts:
           path: ~/testresults/k8s_integration_tests.html
 
+  installer_tests:
+    <<: *executor
+    steps:
+      - checkout
+      - run: |
+          if ! scripts/changes-include-dir deployments/installer tests/packaging; then
```

Comment:
Scope the changes to the specific installer script pytest module instead of anything in the packaging test dir.  Apparently the script supports handling filenames instead of just dirs, so perhaps rename the script to `changes-include-path` or something less specific to directories.